docs/architecture/adr-002-docs-structure.md

```diff
@@ -0,0 +1,270 @@
+- **TECHNICAL-SETUP.md** - how to get started with the repo (could be a link to a tutorial or How-To)
+    - Links to specific tooling set-up - development tools, linters etc
+    - Dependencies such as [pre-commit](https://pre-commit.com/), 
+    - Building the code
+    - Running tests
+- **CONTRIBUTING.md** - This details how to new users can contribute to the project. In specific,
+    - Committing changes
+    - Commit message formats (see below)
+    - Raising PR's
+    - Code of Conduct
+- **CODEOWNERS** - although not part of documentation it will define the code maintainers who responsible for quality assurance on comments, PR's and issues.
+
+#### Modules
+
+In line with Cosmos SDK convention (TODO: needs reference) each module contains its own relevant documentation:
+
+- **Module specifications** - these are document that outline state transitions x/module-name/docs/
+- **Module-level README.md** e.g. x/module-name/README.md
+
+This will be classed as reference documentation. It SHOULD BE descriptive, but explanatory. Explanations should be part of issues, Pull Requests and/or docs/explanation/architecture.
+
+#### docs/
+
+At docs level, this folder shall include the following files and folders:
+
+- **README.md** - SHALL USE this for introduction and orientating the user, based on the content of this ADR and other materials.
+- **CODEOWNERS** - This [CodeOwners file](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners) details the reviewers for documentation folder. This SHALL INCLUDE the code maintainers in the root CODEOWNERS file plus a member of the Tendermint Technical Writing Team.
+
+#### docs/Reference
+
+Reference documentation includes a number of different forms:
+
+- **README.md** - This document would outline the purpose of the reference documentation as per the methodology above. In addition it will also links to documentation created from the code itself, specifically
+    - Code Documentation in form of Go Docs
+    - Swagger API documentation
+- **GLOSSARY.md** - It is key that this is reviewed regulary and applied consistently. These form the terms of reference for users and ensure that discussion and design are based on consistent terms of reference. This file will be similar to [Cosmos Network Glossary](https://v1.cosmos.network/glossary) and can reference this.
+- **MODULES.md** - A markdown document that has references to module relevant documentation
+
+##### docs/Reference/use-cases
+
+This will be a folder that describe Cosmos Cash use cases. Ideally these will be in BDD format. As perscribed above, this information should be dry in nature and avoid explanation the should be cover in the explanation documentation
```

Comment:
how about using a self-describing naming convention for the use case reference docs? IMO, subfolders might turn out to be more trouble than we need. 
```suggestion
This will be a folder that describe Cosmos Cash use cases. Ideally, use cases are written in behavior-driven development (BDD) format. Use case content should be dry in nature and avoid explanation that should be covered in the explanation documentation.
```